### ISSUE TITLE
Add `openssh` to `systems` umbrella package

### ISSUE THREAD

**Issue by dingp** [DUNE-DAQ/daq-release#142]
When testing `nanorc` on `np04-srv-013`, I found that the system `ssh` under `/usr/bin/ssh` was built with an older version of `openssl`. But the new version of `openssl` brought in by loading the spack release is not backward compatible. This resulted in ssh to fail with errors like `ssh: symbol lookup error: ssh: undefined symbol: EVP_KDF_ctrl, version OPENSSL_1_1_1b`. 

This is a similar issue happened to `krb5`. The solution was to add `openssh` to the spack release so it can be built with the newer `openssl`.

**Comment by dingp** [comment]
The issue has been fixed. The following steps has been taken:

1. Commit c5b4fb95ce485e0cc6059c16e4b8844b6046e6b0 added `openssh` to the release YAML.
2. Modify the spack repo for `dunedaq-v2.10.2` and `dunedaq-v2.10.1` and `N22-04-19` by adding `openssl` to `systems` dependency list;
3. For each of the release, do `spack uninstall --dependents systems@<RELEASE_NAME>`;
4. For each of the release do `spack add dunedaq@<RELEASE_NAME>%gcc@8.2.0` followed by `spack concretize --reuse` and `spack install`.

**Comment by dingp** [comment]
Done. Verified the spack release `dunedaq-v2.10.2` and `N22-04-19` works now on `np04-srv-013` with `ssh` from the spack installation.

**Comment by dingp** [comment]
`openssh` in the current `speck-externals` instance does not support kerberos. It is likely due to the fact it was built within a minimal docker image which may not have some system packages installed.

**Comment by dingp** [comment]
Commit c34267c19fce7bd9c1ae0f3b4fbfabaef4fd1dfa adds the `-with-kerberos5` option to `openssh` and additional dependency of `krb5`.

Removed old `openssh` and reinstalled it with these changes. Also reinstalled the umbrella package `systems` in `dunedaq-v2.10.1` `dunedaq-v2.10.2` `N22-04-19` and `N22-04-20` releases.

I've verified that ssh from spack now works with Kerberos on `iceberg01.fnal.gov` (which only allows Kerberized logins). Though I had to have my `~/.ssh/config` with `GSSAPIAuthentication yes` as it is not turned on by default.
